.env.example

```diff
@@ -21,7 +21,9 @@ DEVELOPMENT=True
 
 PROMETHEUS_MULTIPROC_DIR='/tmp'
 CURRENCY_URL=https://open.er-api.com/v6/latest/USD
-UNLEASH_TOKEN='*:*.dbffffc83b1f92eeaf133a7eb878d4c58231acc159b5e1478ce53cfc'
+UNLEASH_ADMIN_TOKEN='*:*.dbffffc83b1f92eeaf133a7eb878d4c58231acc159b5e1478ce53cfc'
```

Comment:
Quick question: why do we need the old token here?

Reply:
No real reason other than "just in case".

Reply:
It's needed for doing certain tasks with the Unleash API. It's good to have if we need it.